js/main.js

```diff
@@ -1,36 +1,49 @@
 
-// runs this function once all DOM has been loaded
+
 window.onload = () => {
+    // selects elements function
+    const selectElement = (element) => document.querySelector(element);
+    const input = selectElement("#tntsearch-wrapper > input[type=text]");
+    const clearSearch = selectElement(".tntsearch-clear");
+    const overlay = selectElement("#overlay");
+    const toggleNav = selectElement("#sidebar-toggle");
+    let bool = false;
 
-    // variable declaration for query selectors
-    const input = document.querySelector("#tntsearch-wrapper > input[type=text]");
-    const clearSearch = document.querySelector(".tntsearch-clear");
-    const clickOverlay = document.querySelector("#overlay");
+    toggleNav.addEventListener("click", () => {
+
+        if (!bool){
```

Comment:
your reading bool here which as far as I can tell will always be false. I'm not sure this code is going to run how you expect it to.

Reply:
I was meant to actually give the bool a proper name i forgot ill change that. But the code does work how i want it as if the bool is false it means that when the user presses the sidebar toggle it will open up the sidebar and the bool then changes to true. Next time the user presses the sidebar toggle the bool will change to false and the sidebar will hide. Ive tested the code and it works perfectly